tox.ini

```diff
@@ -24,7 +24,7 @@ deps =
     sphinx71: Sphinx>=7.1,<7.2
     sphinx72: Sphinx>=7.2,<7.3
     sphinxlatest: Sphinx
-    dev: https://github.com/sphinx-doc/sphinx/archive/refs/heads/master.zip
+    sphinxdev: https://github.com/sphinx-doc/sphinx/archive/refs/heads/master.zip
```

Comment:
Not sure I follow this change. Isn't the target of the `envlist` still `sphinxdev`?

That is: `py311-sphinxdev-qa` not `py311-dev-qa`

Reply:
Ah I understand what is happening here now, this does look correct